gensrc/script/functions.py

```diff
@@ -1212,6 +1212,40 @@
 
     [150335, 'array_sortby', True, False, 'ANY_ARRAY', ['ANY_ARRAY', 'ANY_ARRAY', 'ANY_ARRAY', "..."], 'ArrayFunctions::array_sortby_multi'],
 
+    [150340, 'array_repeat', True, False, 'ARRAY_DATE', ['DATE', 'INT'], 'ArrayFunctions::repeat'],
```

Comment:
```suggestion
    [150340, 'array_repeat', True, False, 'ANY_ARRAY', ['ANY_ELEMENT', 'INT'], 'ArrayFunctions::repeat'],
```
because the cpp function is the same 'ArrayFunctions::repeat', it is not necessary to enumerate all these types.
You can register just one.

Reply:
ok

Reply:
already fixed